README.md

```diff
@@ -88,26 +88,27 @@ For API documentation go to
 
  * [Realm Dart API Docs](https://pub.dev/documentation/realm_dart/latest/)
 
-For a complete documentation go to [Realm Flutter and Dart SDK Docs](https://docs.mongodb.com/realm/sdk/flutter/).
+Use [realm](https://pub.dev/packages/realm) package for Flutter and [realm_dart](https://pub.dev/packages/realm_dart) package for Dart applications.
 
+For a complete documentation go to [Realm Flutter and Dart SDK Docs](https://docs.mongodb.com/realm/sdk/flutter/).
 
-## Limitations
+Read [Quick Start - Flutter SDK]( https://www.mongodb.com/docs/realm/sdk/flutter/quick-start/) if you are running the Realm for the first time.
 
-* It provides the functionality for creating, retrieving, querying, sorting, filtering, updating Realm objects.
+Read more about using the Realm with [Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and connecting to [Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/).
 
-* Migrations are not supported yet.
+## Limitations
 
-    If you change your data models often and receive a migration exception be sure to delete the old `default.realm` file in your application directory. It will get recreated with the new schema the next time the Realm is opened.
+* Data types Set and Map are still not supported.
 
```

Comment:
suggestion: might also want to add other unsupported features.

for example, on the app Services side, MongoDB Data Access isn't supported. 

not sure what else is missing versus what a Realm SDK _should_ have, but here'd be a good place to mention.

Reply:
We have this only: https://www.mongodb.com/docs/realm/sdk/flutter/#release-candidate-limitations.

Reply:
@blagoev commented: "Do we have such limitations page in the docs? We could link to that from here instead?"

Reply:
We have this only: https://www.mongodb.com/docs/realm/sdk/flutter/#release-candidate-limitations. But it doesn't list what is still not implemented. Maybe I have to delete this section. What do you think?

Reply:
i think we can delete this section, then. 

There's no official list of features that a Realm SDK 'should' have, as far as I know. 

Plus, this will be annoying to update as features get added.